js/output/sql/sql-output.js

```diff
@@ -60,105 +60,140 @@ window.SQLOutput = Backbone.View.extend({
         } else if (statement.indexOf("INTERGER") !== -1) {
-            errorMessage += ". " +
-                i18n._(" Is INTEGER spelled correctly?");
+            errorMessage += " " +
+                i18n._("Is \"INTEGER\" spelled correctly?");
         } else if (isSyntaxError &&
                 statement.indexOf("CREATE") !== -1 &&
                 statement.search(/CREATE TABLE \w+\s\w+/) > -1) {
-            errorMessage += ". " +
+            errorMessage += " " +
                 i18n._("You can't have a space in your table name.");
         } else if (isSyntaxError &&
                 statement.indexOf("CREATE TABLE (") > -1) {
-            errorMessage += ". " +
+            errorMessage += " " +
                 i18n._("Are you missing the table name?");
         } else if (isSyntaxError &&
                 statement.indexOf("PRIMARY KEY INTEGER") !== -1) {
-            errorMessage += ". " +
-                i18n._("Did you mean to put PRIMARY KEY after INTEGER?");
+            errorMessage += " " +
+                i18n._("Perhaps you meant to put \"PRIMARY KEY\" after \"INTEGER\"?");
         } else if (isSyntaxError &&
                 statement.indexOf("(") !== -1 &&
                 statement.indexOf(")") === -1) {
-            errorMessage += ". " +
+            errorMessage += " " +
                 i18n._("Are you missing a parenthesis?");
         } else if (isSyntaxError &&
-                statement.indexOf("CREATE") !== -1 && 
+                statement.indexOf("CREATE") !== -1 &&
                 statement.indexOf("TABLE") === -1 && (
                     statement.indexOf("INDEX") === -1 ||
                     statement.indexOf("TRIGGER") === -1 ||
                     statement.indexOf("VIEW") === -1)) {
-            errorMessage += ". " +
+            errorMessage += " " +
                 i18n._("You may be missing what to create. For " +
-                    "example, CREATE TABLE...");
+                    "example, \"CREATE TABLE...\"");
         } else if (isSyntaxError &&
                 statement.indexOf("UPDATE") !== -1 &&
                 statement.indexOf("SET") === -1) {
-            errorMessage += ". " +
-                i18n._("Are you missing the SET keyword?");
+            errorMessage += " " +
+                i18n._("Are you missing the \"SET\" keyword?");
         } else if (isSyntaxError &&
                 statement.search(/[^SUM]\s*\(.*\)\n*\s*\w+/) > -1 ||
                 statement.search(/\n+\s*SELECT/) > -1 ||
                 statement.search(/\)\n+\s*INSERT/) > -1
                 ) {
-            errorMessage += ". " +
+            errorMessage += " " +
                 i18n._("Do you have a semi-colon after each statement?");
         } else if (isSyntaxError &&
             statement.indexOf("INSERT") !== -1 &&
-            statement.search(/[^INSERT],\d*\s*[a-zA-Z]+/) > -1 
+            statement.search(/[^INSERT],\d*\s*[a-zA-Z]+/) > -1
```

Comment:
Hm, it matches the desired text cases, when INSERT is at the start of the line (even if its the second line). I don't want to mess with it in case there was a reason I used the ^ .

Reply:
But the square brackets change the caret from a start of line anchor to a NOT operator and the result is that you are matching any one character that is not one of I, N, S, E, R, or T followed by a comma, optional digits, optional spaces and letters. So "blah INSERT Q,a" matches but "blah INSERT R,a" does not match

Reply:
![image](https://user-images.githubusercontent.com/516914/45898443-d74db580-bd8e-11e8-8feb-7d7769b0309f.png)

Reply:
:-)